### DISCUSSION TITLE
QCefView如何设置透明背景？

### DISCUSSION THREAD

**Discussion by ChenXuyuan97** [CefView/QCefView#176] [Q&A]
我调用QCefConfig，setBackgroundColor两者的setBackgroundColor，发现背景都无法变透明，始终是有点灰度的白色。请问要如何设置，才能将背景变为透明呢？

**Comment by tishion** [comment]
如果使用NCW模式，无法实现。
如果使用OSR模式，可以实现，但是已经不是cef的功能了，需要你自己在paintEvent里实现透明绘制。

**Reply by ChenXuyuan97** [reply]
好的，非常感谢~
